app/app.go

```diff
@@ -440,6 +446,19 @@ func New(
 	app.SetAnteHandler(anteHandler)
 	app.SetEndBlocker(app.EndBlocker)
 
+	upgradeInfo, err := app.UpgradeKeeper.ReadUpgradeInfoFromDisk()
+	if err != nil {
+		panic(err)
+	}
+
+	if upgradeInfo.Name == upgradeName && !app.UpgradeKeeper.IsSkipHeight(upgradeInfo.Height) {
+		storeUpgrades := store.StoreUpgrades{
+			Added: []string{authz.ModuleName, feegrant.ModuleName, wasm.ModuleName},
```

Comment:
```suggestion
			Added: []string{authz.ModuleName, feegrant.ModuleName},
```
same as above comment, we shouldn't be using anything from wasm module